src/main/java/org/datadog/jenkins/plugins/datadog/model/BuildData.java

```diff
@@ -416,6 +428,7 @@ public Map<String, Set<String>> getTags() {
         }
         allTags = TagsUtil.merge(allTags, tags);
         allTags = TagsUtil.addTagToTags(allTags, "job", getJobName("unknown"));
+        allTags = TagsUtil.addTagToTags(allTags, "datadog.product", "cipipeline");
```

Comment:
Nit: Constants candidates?

Reply:
The rest of the code doesn't use constants for things like this, so I kept it consistent. I don't have a strong opinion, though, happy to change this if you prefer it to be a constant.

Reply:
I guess we could do it in a different PR.